zookeeper-server/src/main/java/org/apache/zookeeper/KeeperException.java

```diff
@@ -404,7 +406,9 @@ public void setCode(int code) {
         /** The session has been closed by server because server requires client to do SASL authentication,
          *  but client is not configured with SASL authentication or configuted with SASL but failed
          *  (i.e. wrong credential used.). */
-        SESSIONCLOSEDREQUIRESASLAUTH(-124);
+        SESSIONCLOSEDREQUIRESASLAUTH(-124),
+        /** Operation was throttled and not executed at all. please, retry! */
```

Comment:
It might be better to add more document here on how to deal with this new error code. `please, retry!` does not sound a good answer, as our intention, according to the design is to not to let clients retry the requests immediately. Something like - "this error code indicates that zookeeper server is under heavy load and can't process incoming requests at full speed; please retry with back off" sounds more useful to user.

Reply:
Thanks!